sdk/kiosk/src/types/transfer-policy.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ObjectOwner } from '@mysten/sui.js';
+
+/** The Transfer Policy module. */
+export const TRANSFER_POLICY_MODULE = '0x2::transfer_policy';
+
+/** Name of the event emitted when a TransferPolicy for T is created. */
+export const TRANSFER_POLICY_CREATED_EVENT = `${TRANSFER_POLICY_MODULE}::TransferPolicyCreated`;
+
+/** The Transfer Policy Type */
+export const TRANSFER_POLICY_TYPE = `${TRANSFER_POLICY_MODULE}::TransferPolicy`;
+
+/** The Transer Policy Rules package address on testnet */
+export const TESTNET_RULES_PACKAGE_ADDRESS =
+  'bd8fc1947cf119350184107a3087e2dc27efefa0dd82e25a1f699069fe81a585';
+
+/** The transfer policy rules package address on mainnet */
+export const MAINNET_RULES_PACKAGE_ADDRESS =
```

Comment:
Maybe we should move envs somewhere else.

Reply:
@damirka A constants file on the root dir maybe?